sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -629,12 +629,12 @@ object Radar {
      */
     @JvmStatic
     fun getLocation(callback: RadarLocationCallback? = null) {
-        this.logger.i("getLocation()", RadarLogType.SDK_CALL)
         if (!initialized) {
             callback?.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
 
             return
         }
+        this.logger.i("getLocation()", RadarLogType.SDK_CALL)
```

Comment:
Do we need to do this same pattern on ios?

Reply:
No: https://radarlabs.slack.com/archives/C02MF6PAWBH/p1679951952294969